Dashboard/app/js/lib/views/surveys/question-view.js

```diff
@@ -512,7 +512,7 @@ FLOW.QuestionView = FLOW.View.extend({
         if (isUnique) {
           args.success();
         } else {
-          args.failure('the question id is not unique');
+          args.failure(Ember.String.loc('_question_id_not_unique'));
```

Comment:
May I suggest we rename the placeholders as well in order to keep terms consistent... i.e. `_variable_name`